src/protocol/boolean/bitwise_lt.rs

```diff
@@ -135,11 +135,12 @@ impl BitwiseLessThan {
         a: &[Replicated<F>],
         b: &[Replicated<F>],
     ) -> Result<Replicated<F>, Error> {
-        debug_assert_eq!(a.len(), b.len(), "Length of the input bits must be equal");
-        let mut e = Self::step1(a, b, ctx.narrow(&Step::AXorB), record_id).await?;
+        let (a, b) = align_bit_lengths(a, b);
```

Comment:
Edit `BitwiseLessThan` to work with inputs with different bit-lengths. `BitDecomposition` needs this.